polaris-react/src/components/TextField/TextField.scss

```diff
@@ -264,7 +216,6 @@ $spinner-icon-size: 12px;
   .Input,
   .Backdrop {
     border: none;
-    line-height: var(--p-font-line-height-1);
```

Comment:
Removed this line as it was already being overridden in pre-uplift due to specificity. 
This particular line height declaration also doesn't seem to make sense in the context of the new DL, taking into account the se23 changes that have been made.